recipes-bsp/isp-imx/basler-camera_4.2.2.13.0.bb

```diff
@@ -2,14 +2,14 @@
 
```

Comment:
Changlog please.

Reply:
The Changelog I can see is just 'drop from VSI', no detail.